<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pay-Manager</string>
  </property>
  <property name="toolTip">
   <string>Add new</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>402</height>
    </rect>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;支出管理系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>-1</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="smallButton">
    <property name="geometry">
     <rect>
      <x>604</x>
      <y>-1</y>
      <width>28</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListCtrl" name="qListCtrl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>649</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>385</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;Current Time:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="CTime_Label">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>387</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:600;
color:#FFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>385</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;WwW.GkC.AsIa By: Jakes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Add new</string>
    </property>
    <property name="whatsThis">
     <string>Add new</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="editBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Edit exist</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="delBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Delete</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>387</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:600;
color:#FFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QListCtrl</class>
   <extends>QTableView</extends>
   <header>qlistctrl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smallButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>611</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>edit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qListCtrl</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>edit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>delItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addNew()</slot>
  <slot>edit()</slot>
  <slot>delItem()</slot>
 </slots>
</ui>
